Monika After Story/game/script-topics.rpy

```diff
@@ -2353,7 +2353,22 @@ label monika_rain_holdme:
 
         call monika_holdme_end
         # small affection increase so people don't farm affection with this one.
-        $ mas_gainAffection(modifier=0.25)
+        python:
+            # Monika likes being held for long periods of time, the longer the better
+            mod = 0.25
+            if elapsed_time <= datetime.timedelta(minutes=10):
```

Comment:
`elapsed_time` is undefined.

Reply:
`elapsed_time` doesn't resolve to undefined. In `monika_holdme_reactions`, it gets defined. And apparently due to the way Ren'Py handles Python one-liners and python blocks, it never gets undefined.

Reply:
`elapsed_time` is a common name, thus it can be deleted, cleaned, redefined. It also means if we change it in one place where it's defined, we'd have to change in all other places. We should never use variables from other contexts. Sadly, we can't fix it in this PR, so just add a TODO to change that later.

Reply:
Okay. I'll add that TODO and compute it myself then. :+1:

Reply:
We already thought about it, right now there's no better solution available. So you can keep this global. I need a TODO to not forget about it and fix later (I'm changing this code too).

Reply:
I already did the change on my local repo. Oh well. Makes it so that when the change comes around that it's one less thing that might need fixing. :shrug:

Reply:
Well according to @jmwall24 Doing it the way I was has it's own issues, so looks like I _have to use_ `elapsed_time` in order to get an accurate number. :-/